jpo-ode-common/src/main/java/us/dot/its/jpo/ode/util/XmlUtils.java

```diff
@@ -253,4 +254,8 @@ public static XmlMapper getStaticXmlMapper() {
     return staticXmlMapper;
   }
 
+  public static XmlMapper getPlainXmlMapper() {

```

Comment:
Used in `OdeMessageFrameDataCreatorHelper` to serialize decoded XER messages from the ADM.